include/zephyr/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -210,6 +210,7 @@ SECTIONS
 	__rodata_region_end = .;
 	MPU_ALIGN(__rodata_region_end - ADDR(rom_start));
 	__rom_region_end = .;
```

Comment:
Why is this line still here ?

Reply:
yep thanks, I've updated in the latest push.